gitgud/levels/rampup/_reversing/test.spec

```diff
@@ -1 +1,9 @@
+1 (master, local)
+3
+4 : 3
+5 : 3
+6 : 4 5
+-5
+-3 (remote)
```

Comment:
Do you think you can work on making this syntax available by changing the parser code?

Reply:
sure

Reply:
Hmm do you think we need to make a "git gud revert" that names the revert commit for you? The test parser picks up "-#" just fine, but in the level the commit name is whatever message you write for the revert. (i.e. "Revert 5 \n\nThis reverts commit <hash>")

Reply:
So I think the best way to do this is to look at the commit object itself and just recognize it as deleting file 5 for `-5` and 3 for `-3`, etc. It doesn't matter that it's a revert. A revert is nothing special other than that git is able to construct the commit itself

Reply:
I honestly think we should do that for normal commits too, but it's obviously much easier to just look at the commit name.